net.labymod.LabyMod.metainfo.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>net.labymod.LabyMod</id>
+  
+  <name>LabyMod</name>
+  <summary>All-in-one Minecraft Client</summary>
+  
+  <metadata_license>MIT</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  
+  <description>
+    <p>
+      -- Incredible performance -- Freezes, lags, and poor FPS are a thing of the past: With LabyMod you will achieve the best performance and make your game feel smoother than ever before.
+    </p>
+    <p>
+      -- Your Game. Your Style -- It&apos;s up to you! Switch between a stunning, modern theme in high resolution or the classic and iconic Minecraft style at any time.
+    </p>
+    <p>
+      -- VoiceChat in Minecraft:&#xA; -- Meet new friends or chat with familiar faces - with our VoiceChat you will have even more fun in-game. Our ready-to-go VoiceChat addon can be used on any multiplayer server without any additional setup required.
+    </p>
+    <p>
+      -- Ready for more? -- That&apos;s by far not all. LabyMod offers an improved server list, interactive screenshots, in-game widgets, friends, dozens of add-ons, support for Fabric and Forge and much more! Take a look at the website or download it now.
+    </p>
+    <p>
+      NOTE: This wrapper is not verified by, affiliated with, or supported by LabyMedia GmbH. LabyMod is not an official Minecraft product. It is not approved by or associated with Mojang or Microsoft.
+    </p>
+  </description>
+  
+  <launchable type="desktop-id">net.labymod.LabyMod.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://labymod.net/page/tpl/assets/images/four/launcher.png</image>
+      <caption>LabyMod Launcher</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://laby.net/api/v2/post/image/3924209e-1f3c-4603-b196-41cc1f895fc7.avif</image>
+      <caption>Ingame</caption>
+    </screenshot>
+  </screenshots>
+  <url type="bugtracker">https://labymod.net/ideas</url>
+  <url type="help">https://labymod.net/support</url>
+  <url type="homepage">https://labymod.net/</url>
+  <developer_name>LabyMedia GmbH</developer_name>
+  <developer id="tld.vendor">
```

Comment:
tld.vendor is a template, you replace it with the actual reverse domain...

Reply:
forgot to remove the tag, I added it only for testing purposes. Thx :)